resources/assets/js/app/views/votes/ShowVote.js

```diff
@@ -13,13 +19,57 @@ module.exports = Marionette.LayoutView.extend({
         answers: '#answers'
     },
     ui: {
-        c_count: '#count'
+        c_count: '#count',
+        newCommentButton: '.new-comment-notification'
     },
+
+    events: {
+        'click @ui.newCommentButton': 'showNewComments'
+    },
+
+
+    initialize: function () {
+        this.listenTo(Radio.channel('votesChannel'), 'setCommentsCount' + this.options.voteModel.id, function (n) {
+            this.ui.c_count.text(n + ' Comments');
+        });
+
+        socketCommentClient.bind('VoteComments', this.options.voteModel.id);
+    },
+
+    onBeforeDestroy: function () {
+        this.stopListening();
+        socketCommentClient.unbind('VoteComments', this.options.voteModel.id);
+    },
+
+    onShow: function () {
+        this.addedCommentsCollection = new CommentsCollection([], {parentUrl: ''});
+
+        var self = this;
+        this.collection.listenTo(Radio.channel('VoteComments'),
+            'newComment', function (comment) {
+                self.addedCommentsCollection.add(new CommentModel(comment), {parentUrl: ''});
+                var count = self.addedCommentsCollection.length + self.collection.length;
+                Radio.trigger('votesChannel', 'setCommentsCount' + self.options.voteModel.id, count);
+
+                if (comment.user_id != currentUser.id) {
+                    self.$('.new-comment-notification').show(300);
+                } else {
+                    self.options.collection.add(self.addedCommentsCollection.toJSON());
+                }
+            });
+    },
+
     collectionEvents: {
         'update': function () {
             this.ui.c_count.text(this.collection.length + ' Comments');
```

Comment:
' Comments' is static part. So I guess that it can be placed in template. Or you can update it to dynamic to show 'Comment' for length == 1 and 'Comments' for length > 1